fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -991,13 +1005,29 @@ public Builder setIncreaseLimitAfter(int increaseLimitAfter) {
                 return this;
             }
 
+            /**
+             * Set the time limit. The indexer will exit with a proper exception if this time is exceeded after a
+             * non-final transaction.
+             * @param timeLimitMilliseconds the time limit, in milliseconds units
```

Comment:
```suggestion
             * @param timeLimitMilliseconds the time limit in milliseconds
```

It looks like this one got overlooked while cleaning up the Javadoc

Reply:
Sure!